<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>480</width>
     <height>360</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_label2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>480</width>
     <height>360</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="track_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string> Tracking</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>431</width>
     <height>104</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>H_Max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="h_max">
        <property name="maximum">
         <number>179</number>
        </property>
        <property name="value">
         <number>179</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>S_Max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="s_max">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>V_Max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="v_max">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>431</width>
     <height>104</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>H_Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="h_min">
        <property name="maximum">
         <number>179</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>S_Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="s_min">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>V_Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="v_min">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="stop_track_button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string> Stop Tracking</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cameraSelect">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>76</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>./video/merah.mp4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>./video/vid6.mp4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="c1">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>420</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0,0</string>
   </property>
  </widget>
  <widget class="QLabel" name="c2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>450</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0,0</string>
   </property>
  </widget>
  <widget class="QLabel" name="c3">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>420</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0,0</string>
   </property>
  </widget>
  <widget class="QLabel" name="c4">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>450</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0,0</string>
   </property>
  </widget>
  <widget class="QLabel" name="h_min_val">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="s_min_val">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="v_min_val">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>490</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="h_max_val">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>530</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>179</string>
   </property>
  </widget>
  <widget class="QLabel" name="s_max_val">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>560</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>255</string>
   </property>
  </widget>
  <widget class="QLabel" name="v_max_val">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>600</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>255</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>420</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Corner 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>450</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Corner 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>420</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Corner 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>450</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Corner 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="laser_pointer">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>610</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LaserPointer</string>
   </property>
  </widget>
  <widget class="QLabel" name="mouse_pointer_val">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>637</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0,0</string>
   </property>
  </widget>
  <widget class="QLabel" name="mouse_pointer">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>640</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mouse Pointer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_corner">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>630</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Corner</string>
   </property>
  </widget>
  <widget class="QLabel" name="v_max_corner">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>600</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>255</string>
   </property>
  </widget>
  <widget class="QLabel" name="h_min_corner">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>420</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="h_max_corner">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>530</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>179</string>
   </property>
  </widget>
  <widget class="QLabel" name="v_min_corner">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>490</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="s_min_corner">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>450</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="s_max_corner">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>560</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>255</string>
   </property>
  </widget>
  <widget class="QLabel" name="s_max_pointer">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>560</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>255</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_pointer">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>630</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Pointer</string>
   </property>
  </widget>
  <widget class="QLabel" name="h_min_pointer">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>420</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="v_min_pointer">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>490</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="h_max_pointer">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>530</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>179</string>
   </property>
  </widget>
  <widget class="QLabel" name="s_min_pointer">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>450</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="v_max_pointer">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>600</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>255</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_pointer">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>630</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Pointer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_corner">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>630</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Corner</string>
   </property>
  </widget>
  <widget class="QLabel" name="mode_view">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>630</y>
     <width>62</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>corner</string>
   </property>
  </widget>
  <widget class="QLabel" name="mode_view_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>62</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode =&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Status =&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="mouse_track_status">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>idle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c_up">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>480</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>↑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c_right">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>520</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>→</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c_left">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>520</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>←</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c_down">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>560</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>↓</string>
   </property>
  </widget>
  <widget class="QComboBox" name="corner_select">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>480</y>
     <width>76</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>C1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C4</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="corner_adj">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>480</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="corner_adj_save">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>520</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QLabel" name="laser_pointer_val">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>610</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0,0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reset">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>560</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <zorder>start_button</zorder>
  <zorder>stop_button</zorder>
  <zorder>track_button</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>image_label2</zorder>
  <zorder>image_label1</zorder>
  <zorder>stop_track_button</zorder>
  <zorder>cameraSelect</zorder>
  <zorder>c1</zorder>
  <zorder>c2</zorder>
  <zorder>c3</zorder>
  <zorder>c4</zorder>
  <zorder>h_min_val</zorder>
  <zorder>s_min_val</zorder>
  <zorder>v_min_val</zorder>
  <zorder>h_max_val</zorder>
  <zorder>s_max_val</zorder>
  <zorder>v_max_val</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>laser_pointer</zorder>
  <zorder>mouse_pointer_val</zorder>
  <zorder>mouse_pointer</zorder>
  <zorder>save_corner</zorder>
  <zorder>v_max_corner</zorder>
  <zorder>h_min_corner</zorder>
  <zorder>h_max_corner</zorder>
  <zorder>v_min_corner</zorder>
  <zorder>s_min_corner</zorder>
  <zorder>s_max_corner</zorder>
  <zorder>s_max_pointer</zorder>
  <zorder>save_pointer</zorder>
  <zorder>h_min_pointer</zorder>
  <zorder>v_min_pointer</zorder>
  <zorder>h_max_pointer</zorder>
  <zorder>s_min_pointer</zorder>
  <zorder>v_max_pointer</zorder>
  <zorder>load_pointer</zorder>
  <zorder>load_corner</zorder>
  <zorder>mode_view</zorder>
  <zorder>mode_view_label</zorder>
  <zorder>label_11</zorder>
  <zorder>mouse_track_status</zorder>
  <zorder>c_up</zorder>
  <zorder>c_right</zorder>
  <zorder>c_left</zorder>
  <zorder>c_down</zorder>
  <zorder>corner_select</zorder>
  <zorder>corner_adj</zorder>
  <zorder>corner_adj_save</zorder>
  <zorder>laser_pointer_val</zorder>
  <zorder>pushButton_reset</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
